src/templates/pages/styleguide/forms.twig

```diff
@@ -245,6 +245,28 @@
                 {% endblock %}
             {% endembed %}
 
+            {% embed 'styleguide/block' %}
```

Comment:
do `{% embed 'styleguide/block' with {'id': 'messages'} %}` to make the subsection linkable